jena/src/main/java/com/inrupt/client/jena/JenaBodyHandlers.java

```diff
@@ -44,13 +44,12 @@ public final class JenaBodyHandlers {
 
     private static final String CONTENT_TYPE = "Content-Type";
 
-    /**
-     * Populate a Jena {@link Model} with an HTTP response body.
-     *
-     * @return an HTTP body handler
-     */
-    public static Response.BodyHandler<Model> ofModel() {
-        return responseInfo -> responseInfo.headers().firstValue(CONTENT_TYPE)
+    private static Boolean isSuccess(final Response.ResponseInfo responseInfo) {
+        return responseInfo.statusCode() < 300;
+    }
```

Comment:
There are now three of these:
- Here,
- in int RDF4J counterpart and
- [in the SolidClient](https://github.com/inrupt/solid-client-java/blob/e3e9ba52a8902d9d09e5b7af55dd06dffe3690d7/solid/src/main/java/com/inrupt/client/solid/SolidClient.java#L448) (this one is slightly different).

I would love to see only one if you find a way.

Reply:
https://github.com/inrupt/solid-client-java/pull/1157/commits/5f41ca04f15b8620436334a20c66b650fdf7db2e lifts the logic into the API module, and https://github.com/inrupt/solid-client-java/pull/1159/commits/f92ed37b357693e0b02e6dbda9d5b46d8b4caebf uses the shared function here.